demisto_sdk/commands/test_content/execute_test_content.py

```diff
@@ -8,6 +8,8 @@
 from demisto_sdk.commands.test_content.TestContentClasses import (
     BuildContext, ServerContext)
 
+SKIPPED_INTEGRATION_COMMENT = 'The following integrations are skipped and critical for the test'
```

Comment:
Is this message only for skipped integrtions?

Reply:
Yes, in line 72 we check for all skipped integrations, and only in this case we use this comment.

Reply:
I mean, we don't check for tests?

Reply:
What would you say the effort is to put it here as well? And how ugly do you think it will make our code?

Reply:
shouldn't be a big effort and shouldn't make the code too ugly. I believe that a simple check as done for the skipped integrations can be done for the skipped tests. I will add it and then we can see if we want to keep this change.

Reply:
Feels to me that it might take some more effort, If you think it is better to open a dev task for this then let's.

Reply:
Feels to me that it might take some more effort, If you think it is better to open a dev task for this then let's.